src/main/java/ui/PanelMenuCreator.java

```diff
@@ -12,32 +12,25 @@
 import org.apache.logging.log4j.Logger;
 import ui.issuepanel.PanelControl;
 
-import java.util.ArrayList;
-import java.util.List;
+import java.util.*;
 
 import static ui.components.KeyboardShortcuts.CLOSE_PANEL;
 import static ui.components.KeyboardShortcuts.CREATE_LEFT_PANEL;
 import static ui.components.KeyboardShortcuts.CREATE_RIGHT_PANEL;
 
 public class PanelMenuCreator {
 
-    private static final Logger logger = LogManager.getLogger(MenuControl.class.getName());
-
-    public static final String MILESTONE_FILTER_NAME = "milestone:curr sort:status";
-    public static final String MILESTONE_PANEL_NAME = "Current Milestone";
-
-    public static final String ASSIGNEE_FILTER_NAME = "is:open ((is:issue assignee:me) OR (is:pr author:me))";
-    public static final String ASSIGNEE_PANEL_NAME = "Open issues and PR's";
+    private final String lastLoginUsername;
```

Comment:
Can we name this something more specific? It's the username of the current user.